### ISSUE TITLE
API Issue getting block history

### ISSUE THREAD

**Issue by Egg101** [LogBlock/LogBlock#477]
TL;DR: "params.minutes" is not defined even though the API tutorial says it is, so I tried "params.since" and it's glitchy.

I followed the tutorial here:
https://github.com/LogBlock/LogBlock/wiki/API#how-to-get-block-history-from-logblock

What I'm trying to do is get all block history in the world from the past 30 minutes.
One part that I'm having issues with is "params.minutes". Everything else works fine except for that, it thinks it's undefined. It suggested "params.since" so I tried that since that sounds like what I want.
I did some testing on my test server and it doesn't seem to work, I only have block changes from 2-3 hrs ago and it thinks that those are still within the past 30 minutes. Here is the related code:

LogBlock logblock = (LogBlock)getServer().getPluginManager().getPlugin("LogBlock");
QueryParams params = new QueryParams(logblock);
params.bct = BlockChangeType.DESTROYED;
params.limit = -1;
params.since = 30;
params.world = getServer().getWorld(getConfig().getString("world"));
params.needType = true;
params.needData = true;
params.needDate = true;

This, and the code I use the LB info for, runs every 1 minute (in release version it will run every 30 mins, i just put to 1 min for faster testing).

Interesting notes from my tests for reference:
Test 1)
I destroy 2 blocks. I wait 30+ minutes. Still thinks it was destroyed within past 30 minutes, even with a reload.
Test 2)
I destroy 2 blocks. Using Navicat MySQL editor, I edit 1 block-entry to set the date to 2 hours ago. Plugin correctly sees that block wasn't destroyed in the past 30 minutes.
Test 3)
I destroy 2 blocks. I wait 3 hours. Still thinks all blocks were destroyed within past 30 minutes. Using Navicat I change 1 block-entry to put their date back by one hour. Plugin now correctly sees that block wasn't destroyed in past 30 minutes.
Test 4) Same setup as Test 3 except I changed the entry to put the date back by 1 minute. Plugin still incorrectly thinks that block was destroyed within 30 minutes.

Thank you so much and if you need more info I'll be glad to supply!
